src/theme/table/list.js

```diff
@@ -26,7 +26,13 @@ const List = ({
   if ((data.length === 0 || error) && !showHeader) return null;
   return (
     <>
-      <Header data={header} currentSort={currentSort} headerClassName={headerClassName} />
+      {showHeader ? (
```

Comment:
for new changes while its involve with changes from existing source code we can have a flag for hiding the header, its just a way for the backward compatibility! there is a thousand way to do,
I do like the below one, custom header can be empty element
``` js
customHeader ?? (<Header
          data={header}
          currentSort={currentSort}
          headerClassName={headerClassName}
        />)
```
  by the way as this change has all the requirement it's fine, just keep in mind